Engines/Wine/Plugins/DirectDraw renderer/script.js

```diff
@@ -1,4 +1,6 @@
-include("engines.wine.engine.object");
+const Wine = include("engines.wine.engine.object");
+
+include("engines.wine.plugins.regedit");
```

Comment:
This is really interesting. It seems like quite a few plugins were using `regedit` internally without explicitly including it...
In the best case a different previously executed script in the same context included this script. In the worst case this leads to a crash.